### ISSUE TITLE
autosingleton improvement for filenames

### ISSUE THREAD

**Issue by eharris** [mried/beetsplug#8]
The default naming of starting with two digits and a dash is much too specific.
The naming format needs to be configurable, perhaps with a regex?

Examples:
My filenames start with the artists name, so that if I create playlists that mix and match tracks, it's easy to see what the tracks are.
I don't use dashes, I use underscores as separators.

**Comment by mried** [comment]
Right, this is what I use, so I just implemented it this way. Feel free to implement a more flexible solution and raise a PR.
